tools/bpf2c/bpf2c.cpp

```diff
@@ -129,15 +129,33 @@ main(int argc, char** argv)
         auto iter_end = parameters.end();
         std::map<std::string, std::tuple<std::string, std::function<bool()>>> options = {
             {"--sys",
-             {"Generate code for a Windows driver",
+             {"Generate code for a Windows driver with optional output file name",
```

Comment:
Might this be a more helpful visual aid?
"Generate code for a Windows driver with optional output file name. (--sys [out_file_name])"